src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +50,302 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
```

Comment:
not that I am very fond of the name 'substitute', but, it doesn't necessarily have to do with subscriptions.
The only thing the function does (well, will do in the future) is to substitute ${X} for something else ...

It is not **that** generic, as **ContextElement** is input to the function. If that had been a map instead, with key->value, then it would be 100% generic.

However, I would not use the word 'subscription' is the module name. Subscriptions just happens to be the only user to this more generic function. [ not that I can find a better name ... ]